docs/_static/custom.css

```diff
@@ -0,0 +1,60 @@
+.wy-nav-content {
+    max-width: none;
```

Comment:
Please don't do this. See https://github.com/python/python-docs-theme/issues/4. We ended up with upstream Sphinx adding a limit to width of content, because it's a much better UX.

Reply:
I feel very strongly about this, and I don't agree that limiting the width of the content is better UX... I mean you can always resize your browser if you want smaller, and especially for tables, that size feels very limiting.

Reply:
I'll compromise on ``1080px``; 🤷‍♂

Reply:
👀

Can we limit it to somewhere ~120 characters per line?

Reply:
That's roughly 1080px 👍

Reply:
We chatted out of band, and @gaborbernat has strong opinions and does not wish to stick to the defaults of the theme here.

I'm backing out of this theme-related discussion, because I've had enough of those already (see the aforementioned issue related to docs.python.org) and  don't have the mental bandwidth want to have another discussion pushing for  what is already a norm in most documentation / web interfaces for good reason.

Reply:
Also, I've suggested that he remove the 1080px limit, since that's not useful (it's >150 chars vs the ~100 limit suggested in most places) and isn't a viable compromise IMO.

Reply:
<img width="1440" alt="Screenshot 2020-02-04 at 6 20 35 PM" src="https://user-images.githubusercontent.com/3275593/73746396-1a89a800-477b-11ea-80fb-7616c786dc8f.png">

Attaching a screenshot of the current theme, with the 1080px limit applied, on a full-screen Firefox window on my laptop's screen.

Reply:
On my screen, the change is a bit more dramatic 🤔 
![image](https://user-images.githubusercontent.com/690238/73749953-42194a00-4754-11ea-8840-6f23f12b10f2.png)

Reply:
4K vs not-4K :)

Reply:
Slightly reduced width to fit in 120 characters.